### ISSUE TITLE
bug where code assigns to self?

### ISSUE THREAD

**Issue by mvolkmann** [franciscop/translate#5]
See https://github.com/franciscop/translate/blob/master/src/index.js#L43.
The code is
```js
      opts.keys[name] = opts.keys[name];
```
Should that be the following?
```js
      opts.keys[name] = translate[name];
```

**Comment by franciscop** [comment]
You are totally right; that is a bug, but it will only show as a bug in this pattern:

```js
translate.keys = { google: 'abc' };
translate('abc', { keys: { yandex: 'def', engine: 'google' } });
```

It would be very rare to set a global engine key, pass a key a for a second engine and then tell it to use the first one.

Will fix it ASAP with tests though. Feel free to do a PR if you want 😊
